fontmaker/core/ttfparser.go

```diff
@@ -210,21 +210,21 @@ func (t *TTFParser) Parse(filepath string) error {
 		return err
 	}
 	buff := bytes.NewBuffer(data)
-	return t.parse(buff)
+	return t.ParseByReader(buff)
```

Comment:
Oh yeah, then we no longer need to wrap  `data` with `bytes.Buffer` or the whole function could be:

```go
f, err := os.Open(filepath)
if err != nil {
  return err
}
defer f.Close()

return t.ParseByReader(f)
```

Or

```go
data, err := ioutil.ReadFile(filepath)
if err != nil {
    return err
}

return t.ParseFontData(data)
```

Reply:
That's so true... 🙈  Updated with the latter one.